lbrynet/daemon/Daemon.py

```diff
@@ -523,7 +498,9 @@ def create_session(wallet):
             )
             self.startup_status = STARTUP_STAGES[2]
 
-        d.addCallback(create_session)
+            return defer.succeed(True)
+
+        d = create_session(self.wallet)
```

Comment:
`create_session` doesn't need to be a function, you can just return `self.session.setup()` instead of having a callback.

Reply:
:+1: